### ISSUE TITLE
DSP Buffer Size automatically increasing

### ISSUE THREAD

**Issue by ThatZeoMan** [7thbeatgames/adofai#129]
### Please Check

- [X] I searched for the issues, and made sure there were no duplicates.
- [X] I've deleted all the mods and confirmed the issue persists.

### Contact Information

@thatzeogal (Discord)

### Summary

Whenever the main menu is reloaded through any means, the DSP Buffer Size setting is moved up one stage with a notification. This is despite no audible audio errors, or at least ones that I can hear.

### Version

v2.7.0 (Mac)

### Operating System (OS)

macOS Sonoma (M2)

### Platform

Steam

### Device Info

M2 Mac Mini

### Steps to Reproduce

1. Open the game.
2. Reload the main menu via calibration, manually changing the DSP Buffer Size, or exiting a level.
3. The DSP Buffer Size will be automatically increased.

### Expected Behavior

The DSP Buffer Size should remain at the size set.

### Observed Behavior

The DSP Buffer Size increases by 1 level each time the main menu is reloaded.

### Reproduction Rate

100%

### Anything else?

This bug does not occur when loading the main menu the first time simply through launching the game, it only occurs if its the second or higher time the main menu has been loaded in that session.
[Player.log](https://github.com/7thbeatgames/adofai/files/14175195/Player.log)

**Comment by wjdgustn** [comment]
what is your minimum usable dsp buffer size? it increases if audio seems broken.

**Comment by ThatZeoMan** [comment]
I can't test a size of 64 since it always auto-increases, but 128 and above seem to have no audible issues.

**Comment by wjdgustn** [comment]
menu sfx sound works fine in 64? you can check it without leaving setting menu(esc). audio is broken if sfx sound is slower than before

**Comment by ThatZeoMan** [comment]
yes, menu SFX sounds fine. In fact, I can set the size to 64, then enter calibration before it's automatically increased and it sounds perfectly fine.

**Comment by ThatZeoGal** [comment]
https://github.com/7thbeatgames/adofai/assets/67521686/dfe521b8-c8f3-4788-9d6d-35540d8c0069

i know it's been a while, but I just realized i didn't send a video, so i figured I should do that
